jcl/src/jdk.management/share/classes/com/ibm/lang/management/internal/ExtendedMemoryMXBeanImpl.java

```diff
@@ -84,30 +86,45 @@ protected void startNotificationThread() {
 				return thread;
 			};
 
-			Thread notifier = AccessController.doPrivileged(createThread);
+			Thread notifier = java.security.AccessController.doPrivileged(createThread);
 			notifier.start();
 		}
 	}
 
 	/**
 	 * Deprecated. Use getTotalPhysicalMemorySize().
 	 */
-	/*[IF Sidecar19-SE]
+	/*[IF JAVA_SPEC_VERSION > 8]*/
 	@Deprecated(forRemoval = true, since = "1.8")
-	/*[ELSE]*/
+	/*[ELSE] JAVA_SPEC_VERSION > 8 */
 	@Deprecated
-	/*[ENDIF]*/
+	/*[ENDIF] JAVA_SPEC_VERSION > 8 */
 	public long getTotalPhysicalMemory() {
+		/*[IF JAVA_SPEC_VERSION >= 14]*/
+		return osinstance.getTotalMemorySize();
+		/*[ELSE] JAVA_SPEC_VERSION >= 14 */
 		return osinstance.getTotalPhysicalMemorySize();
+		/*[ENDIF] JAVA_SPEC_VERSION >= 14 */
 	}
 
+	/*[IF JAVA_SPEC_VERSION > 8]*/
+	@Deprecated(forRemoval = true, since = "19")
+	/*[ELSE] JAVA_SPEC_VERSION > 8 */
+	@Deprecated
```

Comment:
The deprecation for Java 8 needs to be removed. Also for getUsedPhysicalMemory().

Reply:
For the record, the objection was that Java 8 code that was warning-free before should continue to be so.

Reply:
Updated.